pkg/builder/builder.go

```diff
@@ -422,6 +433,21 @@ func buildHTTPServerSecretVolume(vdb *vapi.VerticaDB) corev1.Volume {
 	}
 }
 
+// buildEmtyDirVolume returns a generic 'emptyDir' volume
+func buildEmtyDirVolume(volName string) corev1.Volume {
```

Comment:
spelling: Emty => Empty
```suggestion
// buildEmptyDirVolume returns a generic 'emptyDir' volume
func buildEmptyDirVolume(volName string) corev1.Volume {
```